### PR TITLE
Switch to Philip's compass and histogram

### PR DESCRIPTION
**Pull request by PhilipMathieu**
I added divs to place the elements on top of eachother and switched the source notebook to my version so that cells could be imported and injected individually.

### CODE REVIEW

**Comment by PhilipMathieu** [comment]
Closes #4

File: docs/index.html [hunk 1]
```diff
@@ -177,17 +177,36 @@ <h2>Strong Industrial Odors</h2>
...
+	new Runtime().module(define, name => {
+		if (name === "viewof basemap") return true;
+		if (name === "container") return new Inspector(document.querySelector("#observablehq-container"));
+		if (name === "map") return true;
+		if (name === "compass") return new Inspector(document.querySelector("#observablehq-compass"));
+		if (name === "brush") return new Inspector(document.querySelector("#observablehq-brush"));
+		if (name === "filter") return true;
+		if (name === "dirs") return true;
+		if (name === "geoData") return true;
+		if (name === "maplibregl") return true;
```
Comment (bergsalex) [inline docs/index.html:208]: Thank you 🙏 ! Filing this for when I have a chance to try it on my own again.

**Review by bergsalex — APPROVED** [review]

File: docs/index.html (hunk 1 above)
Comment (pbogden) [reply docs/index.html:208]: This could be a nice addition, but conflicts need to be resolved.

**Review by SilentMutant — APPROVED** [review]

**Comment by nwebb-roux** [comment]
Resolved the conflict (it was because Philip and I both added new stuff to the head of index.html) and merging the PR

**Pull request merged**